assets/js/components/search/overrides/CDSRecordsResultsListItemDescription.js

```diff
@@ -0,0 +1,51 @@
+// This file is part of CDS RDM
+// Copyright (C) 2025 CERN.
+//
+// CDS RDM is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import _get from "lodash/get";
+import PropTypes from "prop-types";
+import React from "react";
+import { Item } from "semantic-ui-react";
+
+export const CDSRecordsResultsListItemDescription = ({ result, descriptionStripped }) => {
+    const getMetadataField = (path) => _get(result, path, []);
+
+    const cdsReferenceId = getMetadataField("metadata.identifiers")
+        .find((id) => id.scheme === "cds_ref")?.identifier;
+
+    const experiments = getMetadataField("custom_fields.cern:experiments").map((exp) => exp.title.en);
+    const accelerators = getMetadataField("custom_fields.cern:accelerators").map((acc) => acc.title.en);
+
+    const hasMetaData = cdsReferenceId || accelerators.length > 0 || experiments.length > 0;
+    const referenceOrAccelerators = cdsReferenceId || accelerators.length > 0;
+    return (
+        <div>
```

Comment:
```suggestion
        </>
```
<div> can be a problem for semantic styles